src/app/components/AccountDetailLayout/index.tsx

```diff
@@ -50,12 +48,11 @@ function AccountDetailLayout() {
             >
               {account.name}
             </h2>
+            <span>/</span>
+            <span>{t("title1")}</span>
           </div>
-          <div className="flex-row justify-center flex items-center text-gray-500 text-sm dark:text-neutral-500">
-            {account.connector}
```

Comment:
is the only place to see the connector type on the accounts list page now?

Reply:
Yes, that doesn't fit into the header bar anymore. :man_shrugging: